shell/platform/windows/platform_handler_unittests.cc

```diff
@@ -103,6 +104,52 @@ TEST(PlatformHandler, RejectsGettingUnknownTypes) {
       }));
 }
 
+TEST(PlatformHandler, HasStringsCallsThrough) {
```

Comment:
I was NOT able to run these tests locally!  Are we ok if the checks just pass here on Github?  Which unittests.exe file in `out/` is supposed to run the tests in this file?  @dkwingsmt

Reply:
It should be `flutter_windows_unittests` but it looks like it [was never actually added to the build](https://github.com/flutter/engine/blob/master/shell/platform/windows/BUILD.gn#L235-L255). That would be good to fix in this PR.

> Are we ok if the checks just pass here on Github?

In theory, but since it's missing from the build the GitHub checks aren't telling us anything.

Reply:
Ah, I felt like something was up.  Thanks, I'll add it!